docs/source/tested_models.md

```diff
@@ -14,6 +14,7 @@ List of all models for which import/export has been tested with Fabrik.
 * VGG-16 [\[Source\]](https://gist.github.com/ksimonyan/211839e770f7b538e2d8#file-readme-md)[\[Visualise\]](http://fabrik.cloudcv.org/caffe/load?id=20171208113208hjcvb)
 * DeepYeast [\[Source\]](http://kodu.ut.ee/~leopoldp/2016_DeepYeast/code/caffe_model/)[\[Visualise\]](http://fabrik.cloudcv.org/caffe/load?id=20180102135425bzkzy)
 * SpeechNet [\[Source\]](https://github.com/pannous/caffe-speech-recognition)[\[Visualise\]](http://fabrik.cloudcv.org/caffe/load?id=20180102135032ctsho)
+* SENet [\[Source\]](https://github.com/hujie-frank/SENet) [\[Visualise\]](http://fabrik.cloudcv.org/caffe/load?id=20180106091323ectck)
```

Comment:
Can you add a visualization link like it has been done for the others? You will need to load the model in Fabrik, and click "Share" to generate a URL.

Reply:
I see it's been done below – can you copy that over here to keep things consistent?

Reply:
Here is the visualisation link: http://fabrik.cloudcv.org/caffe/load?id=20180106091323ectck